maro/simulator/scenarios/data_center/virtual_machine.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import List
+
+
+class VirtualMachine:
+    def __init__(self, id: str, req_cpu: int, req_mem: int, lifetime: int):
```

Comment:
add docstring for the classes you defined in this scenario.

"""Summary.

Details if needed.

Args:
    ...

Returns:
    if it has
"""

Reply:
same to other classes